lara-typescript/src/interactive-api-client/api.ts

```diff
@@ -218,7 +218,8 @@ export const getAuthInfo = (): Promise<IAuthInfo> => {
 export const getFirebaseJwt = (firebaseApp: string): Promise<IJwtResponse> => {
   return new Promise<IJwtResponse>((resolve, reject) => {
     getInitInteractiveMessage().then(initMsg => {
-      if (initMsg && initMsg.mode === "runtime" && initMsg.hostFeatures?.getFirebaseJwt?.version === "1.0.0") {
+      if (initMsg
+          && initMsg.hostFeatures?.getFirebaseJwt?.version === "1.0.0") {
```

Comment:
Can be simplified to:
```suggestion
      if (initMsg?.hostFeatures?.getFirebaseJwt?.version === "1.0.0") {
```